setup.py

```diff
@@ -0,0 +1,85 @@
+#   Copyright 2022 The PyMC Developers
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import re
+
+from codecs import open
+from os.path import dirname, join, realpath
+
+from setuptools import find_packages, setup
+
+DISTNAME = "pymc_experimental"
+DESCRIPTION = "A home for new additions to PyMC, which may include unusual probability distribitions, advanced model fitting algorithms, or any code that may be inappropriate to include in the pymc repository, but may want to be made available to users."
+AUTHOR = "PyMC Developers"
+AUTHOR_EMAIL = "pymc.devs@gmail.com"
+URL = "http://github.com/pymc-devs/pymc-experimental"
+LICENSE = "Apache License, Version 2.0"
```

Comment:
perhaps add this as well?

Reply:
good call, added

Reply:
Looks like I can't merge, if you don't mind and it looks good @ColCarroll would you do it?